static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -458,6 +458,8 @@ const initYoutubePlayerForElem = (el) => {
             return;
         }
 
+        iframe.style.display = 'block'
```

Comment:
what is this being used for? could we scope it to only be set if its vertical video?